packages/ckeditor5-dev-env/package.json

```diff
@@ -34,6 +34,9 @@
     "node": ">=6.0.0",
     "npm": ">=3.0.0"
   },
+  "bin": {
```

Comment:
Why did you extract translations tasks from `env/index.js`?

Reply:
I could either leave it as is or create a `bin` script (`translations`). I didn't see a point of duplicating `require().*Translation()` in ckeditor5 repo.

This script accepts `collect`, `upload` or `download` param.